tests/baselines/reference/skipJSDocParsing/deprecated.diff

```diff
@@ -0,0 +1,67 @@
+===================================================================
+--- withJSDoc	With JSDoc
++++ withoutJSDoc	Without JSDoc
+@@ -8,9 +8,8 @@
+         "0": {
+             "kind": "FunctionDeclaration",
+             "pos": 0,
+             "end": 46,
+-            "flags": "Deprecated",
```

Comment:
This baseline shows the diff between the two ASTs in JSON format; this seemed like the least bad way to show that this is working.